docs/static_site/src/pages/api/faq/large_tensor_support.md

```diff
@@ -0,0 +1,227 @@
+
+The following are the cases for large tensor usage where you must specify `dtype` as `int64`:
+
+
+* _randint():_
+```
+low_large_value = 2*32*
+*high_large_value = 2*34
+# dtype is explicitly specified since default type is int32 for randint
+a = nd.random.randint(low_large_value, high_large_value, dtype=np.int64)
+```
+* ravel_multi_index()_ and _unravel_index()_:
+```
+x1, y1 = rand_coord_2d((LARGE_X - 100), LARGE_X, 10, SMALL_Y)
+x2, y2 = rand_coord_2d((LARGE_X - 200), LARGE_X, 9, SMALL_Y)
+x3, y3 = rand_coord_2d((LARGE_X - 300), LARGE_X, 8, SMALL_Y)
+indices_2d = [[x1, x2, x3], [y1, y2, y3]]
+# dtype is explicitly specified for indices else they will default to float32
+idx = mx.nd.ravel_multi_index(mx.nd.array(indices_2d, dtype=np.int64),
+                                  shape=(LARGE_X, SMALL_Y))
+indices_2d = mx.nd.unravel_index(mx.nd.array(idx_numpy, dtype=np.int64),
+                                  shape=(LARGE_X, SMALL_Y))
+```
+* _argsort()_ and _topk()_
+
+They both return indices which are specified by `dtype=np.int64`.
+```
+b = create_2d_tensor(rows=LARGE_X, columns=SMALL_Y)
+# argsort
+s = nd.argsort(b, axis=0, is_ascend=False, dtype=np.int64)
+# topk
+k = nd.topk(b, k=10, axis=0, dtype=np.int64)
+```
+* _index_copy()_
+
+Again whenever we are passing indices as arguments and using large tensor, the `dtype` of indices must be `int64`.
+```
+x = mx.nd.zeros((LARGE_X, SMALL_Y))
+t = mx.nd.arange(1, SMALL_Y + 1).reshape((1, SMALL_Y))
+# explicitly specifying dtype of indices to np.int64
+index = mx.nd.array([LARGE_X - 1], dtype="int64")
+x = mx.nd.contrib.index_copy(x, index, t)
+```
+* _one_hot()_
+
+Here again array is used as indices that act as location of bits inside the large vector that need to be activated.
+```
+# a is the index array here whose dtype should be int64.
+a = nd.array([1, (VLARGE_X - 1)], dtype=np.int64)
+b = nd.one_hot(a, VLARGE_X)
+```
+
+## What platforms and version of MXNet are supported ?
+You can use MXNet with large tensor support in the following configuration:
+
+*MXNet built for CPU on Linux (Ubuntu or Amazon Linux), and only for python bindings.*
+*Custom wheels are provided with this configuration.*
+
+These flavors of MXNet are currently built with large tensor support:
+
+1. MXNet for linux-cpu
+2. MXNet for linux_cu100
+
+Large tensor support only works for *forward pass*. 
+Backward pass is partially supported and not completely tested, so it is considered experimental at best.
+
+Not supported:
+
+* GPU and MKLDNN. 
+* Any permutation of MXNet wheel that contains MKLDNN. 
+* Other language bindings like Scala, Java, R,  and Julia.
+
+## Performance Regression: 
+Roughly 40 operators have shown performance regression in our preliminary analysis: Large Tensor Performance (https://docs.google.com/spreadsheets/d/1GpdNquQb71Is5B-li99JDuiLeEZd-eSjHIIowzGrwxc/edit#gid=1325641791)
```

Comment:
Since a Google docs document is not necessarily persistent, can we maybe add a spoiler or something like that (does our mark-up support it?)

Alternatively, we could just add it as attachment

Reply:
@marcoabreu I didn't understand your point about google docs not being persistent. Also, can you tell me how can I add this excel sheet as attachment to the codebase ?

Reply:
@access2rohit How about exporting it to csv, then converting it to md with this tool? http://www.convertcsv.com/csv-to-markdown.htm
Then you can add it to this file.

Reply:
The issue is it's a big csv (400+ lines) so converting this to csv -> md and then adding it to this file will bloat the file (and won't look good with 400 lines)
What would you suggest in that case?

Reply:
> The issue is it's a big csv (400+ lines) so converting this to csv -> md and then adding it to this file will bloat the file (and won't look good with 400 lines)
> What would you suggest in that case?

I thought it was just a list of 40. I see now that it has several sheets too. I'd leave it in excel format. What if you just upload a .xls file? I'm not sure of the best alternative here. If the person that has the file in the google drive deletes their account, the file is gone forever, so I agree it probably shouldn't live there.

Reply:
@aaronmarkham Done!